services/app-web/src/pages/StateSubmission/RateDetails/V2/SingleRateFormFields.tsx

```diff
@@ -226,17 +226,20 @@ export const SingleRateFormFields = ({
             </FormGroup>
             <FormGroup error={Boolean(showFieldErrors('rateProgramIDs'))}>
                 <Label htmlFor={`${fieldNamePrefix}.rateProgramIDs`}>
-                    Programs this rate certification covers
+                    What rates are included in this certification?
                 </Label>
                 <span className={styles.requiredOptionalText}>Required</span>
+                <span className={styles.requiredOptionalText}>
+                    This information will be used to generate the rate name
+                </span>
                 <PoliteErrorMessage>
                     {showFieldErrors('rateProgramIDs')}
                 </PoliteErrorMessage>
                 <ProgramSelect
```

Comment:
`ProgramSelect` currently shows all programs not just rate programs. I saw code that toggles the component to show only contract programs. I think we need to update it to allow it to also show only rate programs. I don't see a ticket left in the epic to do that, would you be able to add that in this one?

Reply:
One more. On the rate dashboard, old submissions are not displaying the historical programs.
<img src="https://github.com/Enterprise-CMCS/managed-care-review/assets/98117700/1c1276b8-7517-44d6-b40d-1f08720698c6"/>

Reply:
> I think we need to update it to allow it to also show only rate programs. I don't see a ticket left in the epic to do that, would you be able to add that in this one?

@JasonLin0991 Where would this be used? The RateDetails page displays contract + rate programs and the Submission Type page just shows contract programs, I don't believe there's a place where we would only show rate programs

Reply:
Ah, maybe I misunderstood something on the ticket

> Removed AC (duplication of AC in other tickets)
> 1. In the new “what rates are included in this submission” drop down ([Covered by MCR-4043](https://jiraent.cms.gov/browse/MCR-4043))
>    1. If a state has not-programs configured, the list of not-programs are offered
>    2. If a state has no not-programs, the list of overall state-programs are offered

I definitely missed the part where it said that this was `Removed AC (duplication of AC in other tickets)`, sorry.

However, I did check MCR-4043 and it mentions any of those requirements. Did something change?

Reply:
It's a part of this:
- Make sure Rate Details page displays the correct list of programs
- On Contract Details page displays correct list of programs

Reply:
I thought the plan was for rate details to only show rate-programs if any rate-programs exist, otherwise show state-programs

Reply:
@macrael my understanding is that if rate programs exist, display the rate programs and state programs on the rate details page. I had a [slack thread ](https://cmsgov.slack.com/archives/C014CRU197U/p1715907971540969?thread_ts=1715902756.167479&cid=C014CRU197U)with Meghan about it